### DISCUSSION TITLE
Epimodelac2023 - Outbreak Analysis and Modeling in Public Health

### DISCUSSION THREAD

**Discussion by ntorresd** [epiverse-trace/epiverse-trace.github.io#256] [Resources]
### Context

Back in December 2023 the TRACE-LAC team held [_Epimodelac2023  - Curso internacional:
Análisis de Brotes y Modelamiento en Salud Pública_](https://epimodelac.com/). For this course we designed a wide variety of resources with the aims to strengthen the capacity for analysis and modeling of infectious disease outbreaks in the Latin American and Caribbean region, through the use of high-quality, open source and interoperable tools that help in public health decision making. I think these resources are a great place for Spanish speakers to introduce themselves to epidemiological modelling and to get acquainted with some of the packages developed by the TRACE-LAC team.

### URL

https://epiverse-trace.github.io/epimodelac/index.html

### Suggested labels

Modelling, Tutorial

**Comment by avallecam** [comment]
Nice addition! the tutorials are also linked in the Learn tab in the section "tutoriales en español" https://epiverse-trace.github.io/learn.html
